browser/src/VariantPage/VariantSiteQualityMetrics.tsx

```diff
@@ -625,11 +628,29 @@ const SiteQualityMetricsHistogram = ({
 }: SiteQualityMetricsHistogramProps) => {
   const isLogScale = metric === 'SiteQuality' || metric === 'AS_QUALapprox' || metric === 'DP'
```

Comment:
SiteQuality, AS_QUALapprox, and DP aren't currently log scaled on the variant pages as far as I can tell (https://gnomad.broadinstitute.org/variant/1-55051215-G-GA?dataset=gnomad_r4) -- should they be?
<img width="640" alt="image" src="https://github.com/user-attachments/assets/ba19b7b7-12c6-4704-b0bf-2492d98ec00a">

Reply:
Just updated that for the labeling, does the graph look correctly scaled?

Reply:
that the graphs are already log scaled and just needed the labeling additions?

Reply:
ah thanks for clarifying. it seems like the graphs aren't actually log scaled. for that variant ([1-55051215-G-GA](https://gnomad.broadinstitute.org/variant/1-55051215-G-GA?dataset=gnomad_r4)), the browser is showing this for the exomes: 
![image](https://github.com/user-attachments/assets/8bed1b27-b904-4932-ab39-606a21c91170)
the value for AS_QUALapprox in the v4.1 exomes release HT is 262031, and that value is not log transformed

Reply:
<img width="976" alt="Screenshot 2024-09-13 at 10 58 16 AM" src="https://github.com/user-attachments/assets/9746c5be-9967-40cc-a30a-00b42e4f4952">

Using the same variant and view provided this what we have right now after adding the "log()" around the values, would it be better to evaluate or leave it like that with the log labeling?

Reply:
I'm a little confused -- the plot doesn't seem to be log scaled in that second screenshot. The x-axis in the first screenshot (untransformed plot) has the same range as the second log-transformed screenshot, which shouldn't be the case. 

Also, `log10(262031)` is `5.418`, and it looks like that value is now being labeled at the correct location in the second screenshot with the vertical line and arrow. However, in the untransformed plot, 262031 seems to be near the middle of the distribution, and it should stay at the same spot once the rest of the data are transformed (but it is currently far off to the left).

let me know if this doesn't make sense or if I've completely missed something